conf.py

```diff
@@ -75,7 +75,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['reference/technical_information/choose_database.rst']
+exclude_patterns = ['reference/technical_information/choose_database.rst.inc']
```

Comment:
inc means include
files that are meant only to be included, they are not real pages